integrationtest/mapper/pojo-base/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/smoke/ExcludePathsIT.java

```diff
@@ -0,0 +1,291 @@
+											.field( "date", entity2.getEmbedded().getLocalDate() )
+									)
+									.objectField( "customBridgeOnProperty", b3 -> b3
+											.field( "text", entity2.getEmbedded().getEmbedded().getText() )
+											.field( "date", entity2.getEmbedded().getEmbedded().getLocalDate() )
+									)
+									.objectField( "myEmbedded", b3 -> b3
+											.field( "myTextField", entity2.getEmbedded().getEmbedded().getText() )
+											.field( "myLocalDateField", entity2.getEmbedded().getEmbedded().getLocalDate() )
+											.objectField( "customBridgeOnClass", b4 -> b4
+													.field( "date", entity2.getEmbedded().getEmbedded().getLocalDate() )
+											)
+											.objectField( "customBridgeOnProperty", b4 -> b4
+													.field( "text", entity2.getEmbedded().getEmbedded().getEmbedded().getText() )
+													.field( "date", entity2.getEmbedded().getEmbedded().getEmbedded().getLocalDate() )
+											)
+									)
+							)
+					)
+					.add( "3", b -> b
+							.field( "myLocalDateField", entity3.getLocalDate() )
+							.field( "myTextField", entity3.getText() )
+							.objectField( "customBridgeOnClass", b2 -> b2
+									.field( "text", entity3.getText() )
+									.field( "date", entity3.getLocalDate() )
+							)
+							.objectField( "customBridgeOnProperty", b2 -> b2
+									.field( "text", entity3.getEmbedded().getText() )
+									.field( "date", entity3.getEmbedded().getLocalDate() )
+							)
+							.objectField( "myEmbedded", b2 -> b2
+									.field( "myTextField", entity3.getEmbedded().getText() )
+									.field( "myLocalDateField", entity3.getEmbedded().getLocalDate() )
+									.objectField( "customBridgeOnClass", b3 -> b3
+											.field( "date", entity3.getEmbedded().getLocalDate() )
+									)
+									.objectField( "customBridgeOnProperty", b3 -> b3
+											.field( "text", entity3.getEmbedded().getEmbedded().getText() )
+											.field( "date", entity3.getEmbedded().getEmbedded().getLocalDate() )
+									)
+									.objectField( "myEmbedded", b3 -> b3
+											.field( "myTextField", entity3.getEmbedded().getEmbedded().getText() )
+											.field( "myLocalDateField", entity3.getEmbedded().getEmbedded().getLocalDate() )
+											.objectField( "customBridgeOnClass", b4 -> b4
+													.field( "date", entity3.getEmbedded().getEmbedded().getLocalDate() )
+											)
+											.objectField( "customBridgeOnProperty", b4 -> b4
+													.field( "text", entity3.getEmbedded().getEmbedded().getEmbedded().getText() )
+													.field( "date", entity3.getEmbedded().getEmbedded().getEmbedded().getLocalDate() )
+											)
+									)
+							)
+					)
+					.add( "6", b -> b
+							.field( "myLocalDateField", entity6.getLocalDate() )
+							.field( "myTextField", entity6.getText() )
+							.objectField( "customBridgeOnClass", b2 -> b2
+									.field( "text", entity6.getText() )
+									.field( "date", entity6.getLocalDate() )
+							)
+					);
```

Comment:
I hope you'll simplify all this, there seems to be a fair amount of unnecessary complexity :)